.github/workflows/parachain-ci.yml

```diff
@@ -240,7 +240,7 @@ jobs:
           default: true
 
       - name: Run unittests
-        run: cargo test --locked --release -p pallet-* --lib
+        run: cargo test --features=skip-ias-check --locked --release -p pallet-* --lib
```

Comment:
this is really strange, running `pallet-sidechain` alone is fine but it fails with wildcard.

Can you try to investigate what happened? Is it related to the file structure (they are in subdirectories of `pallets`/)

Reply:
Do we still need this feature? Or do we actually need two lines, one with `skip-ias-check` feature, one without

Reply:
Yes, we need.
> enable `skip-ias-check` to run `identity-management` unit tests and 
> disable to run  `pallet-sidechain` 's unit tests